language/stdlib/modules/doc/LibraAccount.md

```diff
@@ -2260,3 +2261,40 @@ also be added. This account will be a child of
 
 
 </details>
+
+<a name="0x1_LibraAccount_Specification"></a>
+
+## Specification
+
+
+Returns field
+<code>key_rotation_capability</code> of the
+LibraAccount under
+<code>addr</code>.
```

Comment:
you don't need to mix html with markdown this way (https://daringfireball.net/projects/markdown/syntax)

Reply:
You are welcome to rewrite docgen as long as you preserve its functionality. There are endless constraints you need to consider. In this example, {backquote} can't be used because we are bolding move keywords and adding links to definitions, which doesn't work inside markdown code blocks. 

It's generally OK to use html in markdown and is common practice, as you find documented in the countless markdown specs on the web.

Reply:
not sure what docgen is, I thought this was an issue with a lack of markdown knowledge :)

it is fine to mix html and markdown, but heavy use of html instead of markdown elements reduces the benefits of markdown.

Reply:
This file is autogenerated from the Move sources. I'll remember to put a disclaimer in the output to avoid such confusion.